.husky/commit-msg

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
-# Section for git-secrets
-if ! command -v git-secrets &> /dev/null
-then
-    echo "git-secrets is not installed. Please visit https://github.com/awslabs/git-secrets#installing-git-secrets"
```

Comment:
Yup we need this to stay in please. I highly doubt this site will ever contain any sensitive keys but it was a requirement made by infosec. Same for the other files too.

Reply:
Hey, @ashleynolan @kevinrodrigues the git secrets checks remain in this pre-commit file: https://github.com/justeattakeaway/pie/pull/44/files#diff-d2bc4bbf14eadc292d84e0ef5f7a93115c23557f533809fc80b896934291529d

The two deletion you see here are the files that husky adds to work with Yarn 2+. By downgrading to Yarn 1 and reconfiguring husky to work with this yarn version, it just means these additional files aren't used anymore. 

If you pull this branch locally you should hopefully see git secrets still running on each commit :)

Reply:
This is why the original implementation of git secrets didn't work for me locally - because it was set up for Yarn 1 like this is, whilst we were using Yarn 2 +

Reply:
Sounds reasonable to me, as long as these checks are getting run pre-commit I'm happy.